downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,39 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- Fixed a bug that resulted in variables that may or may not be initialized in an `on fail` clause not being identified as potentially uninitialized variables.
```

Comment:
I believe we decided to use the format "A bug that resulted in ... has been fixed" (e.g., https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.7.0).

To avoid reading as the bug fix is resulting in the particular behaviour.